### PR TITLE
Moved Health to OpenRA.Mods.Common

### PR DESCRIPTION
**Pull request by huwpascoe**
As part of https://github.com/OpenRA/OpenRA/issues/6820

### CODE REVIEW

**Comment by huwpascoe** [comment]
Fixed stylecop violations

**Comment by huwpascoe** [comment]
fixed visual studio's sneaky removal of final CRLF in csproj.

File: OpenRA.Mods.Common/Traits/Health.cs [hunk 1]
```diff
@@ -210,5 +214,13 @@ public static void InflictDamage(this Actor self, Actor attacker, int damage, IW
 			if (health == null) return;
 			health.InflictDamage(self, attacker, damage, warhead, false);
 		}
+
+		public static void Kill(this Actor self, Actor attacker)
+		{
+			if (self.Disposed) return;
+			var health = self.TraitOrDefault<Health>();
```
Comment (pchote) [inline OpenRA.Mods.Common/Traits/Health.cs]: You can save a trait lookup by keeping Kill on the actor and then using the cached health reference there.

**Comment by huwpascoe** [comment]
Moved the health ext stuff into actor class as requested

**Comment by huwpascoe** [comment]
replaced unwanted spaces with tabs (visual studio is not behaving today)

**Comment by reaperrr** [comment]
Code looks good to me, in-game still works :+1:

**Comment by RoosterDragon** [comment]
:+1: LGTM

**Comment by RoosterDragon** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-%28bleed%29/_compare/2167c77a84a9873932bc20bdea24147dae6b997e...191b3fd2e3c06cb0f74d824659b6ef58cb5e824d)

**Pull request merged**